website/docs/cdktf/create-and-deploy/terraform-cloud.mdx

```diff
@@ -0,0 +1,141 @@
+
+```ts
+import { Construct } from "constructs";
+import { App, TerraformStack, RemoteBackend } from "cdktf";
+
+class MyStack extends TerraformStack {
+  constructor(scope: Construct, name: string) {
+    super(scope, name);
+    // Remote Backend - /docs/backends/types/remote.html
+    new RemoteBackend(this, {
+      // You can omit this if you are not using a self-hosted Terraform enterprise instance.
+      hostname: "app.terraform.io",
+      organization: "company",
+
+      workspaces: {
+        name: "my-app-prod",
+      },
+    });
+
+    // define resources here
+  }
+}
+
+const app = new App();
+const myStack = new MyStack(app, "my-stack");
+// You can also define the backend outside of the stack.
+// new RemoteBackend(myStack, {
+//     hostname: "app.terraform.io",
+//     organization: "company",
+
+//     workspaces: {
+//     name: "my-app-prod",
+//     },
+// });
+// You must configure the RemoteBackend before the app.synth() call
+app.synth();
+```
+
+3. Run `cdktf init` and log in to Terraform Cloud.
+
+### VCS-driven Workflow
+
+Check your synthesized code into version control and connect your Terraform Cloud workspace to that repository with the VCS-driven workflow. Refer to [Deployment Patterns](/cdktf/create-and-deploy/deployment-patterns#deploy-with-terraform-and-terraform-cloud) for more details about how to use CDKTF with the VCS-driven workflow.
+
+## Managing Variables and Secrets through Terraform Cloud
+
+Terraform Cloud [variables](/cloud-docs/workspaces/variables) let you define the variables and secrets that Terraform uses during remote operations. You can set variables specifically for each workspace or you can create variable sets to reuse the same variables across multiple workspaces. For example, you could define a variable set of provider credentials and automatically apply it to all of the workspaces using that provider. 
+
+To use variables, set your workspace to [Remote Execution Mode](/cloud-docs/workspaces/settings#execution-mode). Terraform can only access workspace variables when executing remotely on Terraform Cloud. 
+
+Use the `TerraformVariable` construct to declare variables in your CDKTF application. This creates an undefined [Terraform input variable](/language/values/variables). The following example demonstrates how to create a `TerraformVariable` called `my-var`. 
+
+```ts
+import { Construct } from "constructs";
+import { App, TerraformStack, TerraformVariable } from "cdktf";
+
+class MyStack extends TerraformStack {
+  constructor(scope: Construct, name: string) {
+    super(scope, name);
+    // You can configure the variable 'my-var' in Terraform Cloud and use the value throughout your CDKTF application
+    const value = new TerraformVariable(this, "my-var", {
+      default: "The default value to use",
+      description: "",
+      nullable: false, // if passing no
+      sensitive: true,
+      type: "string",
+    });
+
+    // define resources here
+  }
+}
+```
```

Comment:
Indenting things so that the numbering will work! :-) 

```suggestion
2. Use the `TerraformVariable` construct to declare variables in your CDKTF application. This creates an undefined [Terraform input variable](/language/values/variables). The following example demonstrates how to create a `TerraformVariable` called `my-var`. 

    ```ts
    import { Construct } from "constructs";
    import { App, TerraformStack, TerraformVariable } from "cdktf";

    class MyStack extends TerraformStack {
      constructor(scope: Construct, name: string) {
        super(scope, name);
        // You can define 'my-var' in Terraform Cloud and use the value in your application.
        const value = new TerraformVariable(this, "my-var", {
          default: "The default value to use",
          description: "",
          nullable: false, // if passing no
          sensitive: true,
          type: "string",
        });

        // define resources here
      }
    }
    ```
```